src/style/CharityPage.less

```diff
@@ -66,47 +65,43 @@
 			}
 		}
 	} // ./column
-
-	.top-bands {
-		position: fixed;
-		top: @navbar-height;
-		left: 0;
-		width: 100%;
-		.band1 {
-			background: linear-gradient(to right, #819EB7, #C1DEF2);
-			height: @band-1-height;
-		}
-		.band2 {
-			background: linear-gradient(to right,#ACB09F, #D2D6C3);
-			height: @band-2-height;
-		}
-		.band3 {
-			background: linear-gradient(to right, #819EB7, #C1DEF2);
-			height: @band-3-height;
-		}
-	}
 }
 
 #charity {
 	.impact-column {
 		.header {
-			background: #C1DEF2;
 			.extend-bg-to-sides(@bs-column-padding);
 			padding-top: @band-1-height;
 			margin-bottom: 30px;
 
 			.charity-name {
-				background-color: #D2D6C3;
```

Comment:
Please define colours in `colours.less`, then use variables. Thanks.

Reply:
Done